macos/builder_setup.sh

```diff
@@ -10,15 +10,15 @@ set -e
 # - A MacOS 10.13.6 (High Sierra) box
 
 # About brew packages:
-# We use a custom homebrew tap (DataDog/datadog-agent-macos-build)
+# We use a custom homebrew tap (DataDog/datadog-agent-macos-build, hosted in https://github.com/DataDog/homebrew-datadog-agent-macos-build)
 # to keep pinned versions of the software we need.
 
 # How to update a version of a brew package:
 # 1. See the instructions of the DataDog/homebrew-datadog-agent-macos-build repo
 #    to add a formula for the new version you want to use.
 # 2. Update here the version of the formula to use.
 export PKG_CONFIG_VERSION=0.29.2
-export RUBY_VERSION=2.4.10
+export RUBY_VERSION=2.7.3
```

Comment:
There is a more recent `2.7.x` release
```suggestion
export RUBY_VERSION=2.7.4
```